src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy

```diff
@@ -149,6 +150,31 @@ public class Converter {
         return model[0];
     }
 
+    public static List<ModelASTStep> scriptToPlainSteps(String script) {
```

Comment:
So no complaints here, but I'm wondering if we can make this (and the related changes in `ModelParser` generic - i.e., parameterize the root element for parsing, rather than going into a hardcoded method. Just a thought.